client/styles/style.css

```diff
@@ -1,47 +1,136 @@
 @font-face {
```

Comment:
FYI, it looks like the font used by airbnb is 'Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif'